pkg/controller/apiserver/apiserver_controller.go

```diff
@@ -164,6 +165,21 @@ func (r *ReconcileAPIServer) Reconcile(request reconcile.Request) (reconcile.Res
 		return reconcile.Result{}, err
 	}
 
+	var isManagement = network.Spec.ClusterManagementType == operatorv1.ClusterManagementTypeManagement
+	var tunnelCASecret *v1.Secret
+	if isManagement {
+		tunnelCASecret, err = utils.ValidateCertPair(r.client,
```

Comment:
I think we need a watch added for this secret.

Reply:
@tmjd Added: https://github.com/tigera/operator/pull/626